src/components/SummaryBox/SummaryBox.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import classnames from 'classnames'
+
+interface SummaryBoxProps {
+  heading: string
+  children?: React.ReactNode
+  className?: string
+}
+
+export const SummaryBox = ({
+  heading,
+  children,
+  className,
+  ...props
```

Comment:
What do you think about using `divProps` instead of `props` for the variable name here so that its extra clear what this?

Reply:
Absolutely!